src/metabase/api/dashboard.clj

```diff
@@ -918,7 +919,7 @@
     (get-in card [:dataset_query :native :template-tags (u/qualified-name tag)])))
 
 (defn- param-type->op [type]
-  (if (get-in mbql.s/parameter-types [type :operator])
+  (if (get-in lib.schema.parameter/types [type :operator])
```

Comment:
Originally moved the schemas for parameters into MLv2 so we can add normalization logic as needed. Didn't end up doing this in the end, but previously the MLv2 schema didn't check `:parameters` at all, so this is still a net win